contracts/Web3Entry.sol

```diff
@@ -709,13 +722,40 @@ contract Web3Entry is IWeb3Entry, NFTBase, Web3EntryStorage, Initializable, Web3
         emit Events.SetOperator(characterId, operator, block.timestamp);
     }
 
+    function _approveOperatorList(uint256 characterId, address[] calldata operatorList) internal {
```

Comment:
It seems there could be some common code for both _approveOperatorList and _disapproveOperatorList to share.